client/src/app/containers/Suggestions/slice.js

```diff
@@ -0,0 +1,33 @@
+import { nanoid } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+export const initialState = {
+    handCards: [
+        'AH', 'KH', '5H', 'JH', '8C', '3D',
+    ],
```

Comment:
:mag:  I need a way to update this array

Reply:
Write a reducer below that takes a string and an index to update?